src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -490,12 +488,11 @@ function IOURequestStepConfirmation({
                     />
                     {isLoading && <FullScreenLoadingIndicator />}
                     <View style={[styles.flex1, isLoading && styles.opacity0]}>
-                        {/* @ts-expect-error TODO: Remove this once MoneyRequestConfirmationList (https://github.com/Expensify/App/issues/36130) is migrated to TypeScript. */}
                         <MoneyRequestConfirmationList
                             transaction={transaction}
                             hasMultipleParticipants={iouType === CONST.IOU.TYPE.SPLIT}
                             selectedParticipants={participants}
-                            iouAmount={transaction?.amount}
+                            iouAmount={transaction?.amount ?? 0}
```

Comment:
I don't think this is necessary, unless it was causing a TS error?

```suggestion
                            iouAmount={transaction?.amount}
```

Reply:
It is causing an TS error @youssef-lr, I checked and it is a required prop

Reply:
okay cool! Testing now locally, then I'll merge if nothing comes up